src/components/PlanBrowser/PlanBrowser.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState, type ReactNode } from "react";
+import type { Plan } from "../../utils/api";
+import {
+  Box,
+  Container,
+  Grid2 as Grid,
+  Paper,
+  Typography,
+} from "@mui/material";
+import SearchablePlanList from "./SearchablePlanList";
+
+export type PlanBrowserProps = {
+  plans: Plan[];
+  renderPlan: (plan: Plan) => ReactNode;
+};
+
+export default function PlanBrowser({ plans, renderPlan }: PlanBrowserProps) {
+  const [selectedPlan, setSelectedPlan] = useState<Plan | null>(null);
+
+  return (
+    <Container sx={{ height: "85vh", width: "80%" }}>
+      <Grid container spacing={1} sx={{ height: "100%" }}>
+        <Grid size={{ xs: 4 }}>
+          <Paper elevation={2} sx={{ height: "100%" }}>
+            <SearchablePlanList
+              plans={plans}
+              selectedPlan={selectedPlan}
+              updateSelection={setSelectedPlan}
+            />
+          </Paper>
+        </Grid>
+        <Grid size={{ xs: 8 }}>
+          <Paper
+            elevation={2}
+            sx={{
+              height: "100%",
+              p: 2,
+              display: "flex",
+            }}
+          >
+            {!selectedPlan ? (
```

Comment:
Would make more sense the other way
```suggestion
            {selectedPlan ? (
```